aztk_cli/spark/endpoints/job/get_app_logs.py

```diff
@@ -12,9 +14,19 @@ def setup_parser(parser: argparse.ArgumentParser):
                         dest='app_name',
                         required=True,
                         help='The unique id of your job name')
+    parser.add_argument('--output',
+                        help='Path to the file you wish to output to. If not \
+                              specified, output is printed to stdout')
 
 
 def execute(args: typing.NamedTuple):
     spark_client = aztk.spark.Client(config.load_aztk_secrets())
-    app_logs = spark_client.get_job_application_log(args.job_id, args.app_name)
-    print(app_logs.log)
+    app_log = spark_client.get_job_application_log(args.job_id, args.app_name)
+    if args.output:
+        spinner = utils.Spinner()
```

Comment:
Lines 26 through 30 seem to be used in several places. Worth moving to utils and re-using?